### ISSUE TITLE
DW20 119 1.2.0 Crash

### ISSUE THREAD

**Issue by dalgrim** [FTBTeam/FTB-Modpack-Issues#2623]
### Modpack

FTB Legend of the Eyes

### Modpack version

1.2.0

### Log Files

https://pastebin.com/WtNGvSSq
https://pastebin.com/JvTdf7cw

### Describe the bug

game Client crashes when opening AE wireless terminal and trying to type anything the search field.
Running a dedicated server and the server does NOT crash, only the game client.

### Steps to reproduce

1. Open AE wireless crafting terminal
2. Try to type anything in the search box

### Expected behaviour

I'd expect the game not to crash. It is 100% unplayable as this crash happens every time.

### Screenshots

n/a

**Comment by dalgrim** [comment]
And to add insult to injury the server downgrade is broken so 1.1.1 can't be used either... It's not cleaning up the new mods it installed in 1.2.0 so there are unsupported hanging jars in the mods folder

**Comment by JoshuaAnderson18** [comment]
I'm having the same issue when upgrading my server to v1.2.0 I notice the FTB App crashes from a repeating error:

 java.lang.NullPointerException: Registry Object not present: createplus:goggle_unit

CreatePlus makes it so the goggles can be used as a bauble. In this version of the pack with the mod installed if you try to look up any create items in JEI it does not give any names it just says error look in logs.

I realized the mod wasn't in the server files so I tried adding it but then the server would crash when trying to launch. I looked at the curse forge page and one of the issues posted a week ago about this versions was that it crashes the server but not the client. 

Removing "createplus-1.19.2_v0.5.3-beta.jar" from the client's mod folder seemed to resolve the issue.
